src/main/java/com/denizenscript/clientizen/scripts/containers/gui/elements/LabeledSliderElement.java

```diff
@@ -0,0 +1,72 @@
+package com.denizenscript.clientizen.scripts.containers.gui.elements;
+
+import com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.queues.ContextSource;
+import com.denizenscript.denizencore.tags.ParseableTag;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.denizenscript.denizencore.tags.TagManager;
+import com.denizenscript.denizencore.utilities.YamlConfiguration;
+import com.denizenscript.denizencore.utilities.debugging.Debug;
+import io.github.cottonmc.cotton.gui.widget.WLabeledSlider;
+import io.github.cottonmc.cotton.gui.widget.WWidget;
+import net.minecraft.text.Text;
+
+import java.util.Map;
+
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.getDebugPath;
+import static com.denizenscript.clientizen.scripts.containers.gui.GuiScriptContainer.getTaggedString;
+
+public class LabeledSliderElement implements GuiScriptContainer.GuiElementParser {
+
+    // <--[language]
+    // @name Labeled Slider GUI Element
+    // @group GUI System
+    // @description
+    // Labeled sliders work similarly to <@link language Slider GUI Element>s, but have a label and a different design; they have a UI type of "labeled_slider".
+    //
+    // Labeled sliders have all the same keys and values as normal <@link language Slider GUI Element>s, and additionally:
+    // <code>
+    // ui_type: labeled_slider
+    // # A static unchanging label for the slider, optional.
+    // label: <text>
+    // # A dynamic label for the slider that updates every time its value changes, optional.
```

Comment:
"updates every time its value changes" could imply that it somehow detects when the value changes. Perhaps `updates every frame`

Reply:
I mean, it _does_ update every time the value changes - whenever the slider's value is changed `LibGUI` calls this supplier to get an updated label.